wpimath/src/main/native/include/frc/estimator/ExtendedKalmanFilter.h

```diff
@@ -334,6 +340,43 @@ class ExtendedKalmanFilter {
     m_P = (Eigen::Matrix<double, States, States>::Identity() - K * C) * m_P;
   }
 
+  void InitSendable(wpi::SendableBuilder& builder) override {
+    builder.SetSmartDashboardType("ExtendedKalmanFilter")
+        .AddSmallDoubleArrayProperty(
+            "modelStdDevs",
+            [&](wpi::SmallVectorImpl<double>& values) -> wpi::span<double> {
+              Eigen::Vector<double, States> diag = m_contQ.diagonal();
+              diag.array().pow(0.5);
+              values.assign(diag.data(), diag.data() + diag.size());
+              return values;
+            },
+            [&](wpi::span<const double> stdDevs) {
+              for (size_t i = 0; i < stdDevs.extent; i++) {
+                m_contQ(i, i) = stdDevs[i] * stdDevs[i];
+              }
+            })
+        .AddSmallDoubleArrayProperty(
+            "measurementStdDevs",
+            [&](wpi::SmallVectorImpl<double>& values) -> wpi::span<double> {
+              Eigen::Vector<double, Outputs> diag = m_contR.diagonal();
+              diag.array().pow(0.5);
+              values.assign(diag.data(), diag.data() + diag.size());
+              return values;
+            },
+            [&](wpi::span<const double> stdDevs) {
+              for (size_t i = 0; i < stdDevs.extent; i++) {
+                m_contR(i, i) = stdDevs[i] * stdDevs[i];
+              }
+            })
```

Comment:
This code needs to call the DARE solver to recompute K, which is expensive.

Reply:
It'll only be called if the gains are actually changed, though, which shouldn't happen while the robot is doing anything.

Reply:
This comment should be on KalmanFilter.h instead.